sdk/storage/azqueue/client.go

```diff
@@ -81,46 +82,43 @@ func NewClientFromConnectionString(connectionString string, options *ClientOptio
 		if err != nil {
 			return nil, err
 		}
-		return NewClientWithSharedKeyCredential(parsed.ServiceURL, credential, options)
+		return NewServiceClientWithSharedKeyCredential(parsed.ServiceURL, credential, options)
 	}
 
-	return NewClientWithNoCredential(parsed.ServiceURL, options)
+	return NewServiceClientWithNoCredential(parsed.ServiceURL, options)
 }
 
-func (s *Client) generated() *generated.ServiceClient {
+func (s *ServiceClient) generated() *generated.ServiceClient {
 	return base.InnerClient((*base.Client[generated.ServiceClient])(s))
 }
 
-func (s *Client) sharedKey() *SharedKeyCredential {
+func (s *ServiceClient) sharedKey() *SharedKeyCredential {
 	return base.SharedKey((*base.Client[generated.ServiceClient])(s))
 }
 
-// URL returns the URL endpoint used by the Client object.
-func (s *Client) URL() string {
+// URL returns the URL endpoint used by the ServiceClient object.
+func (s *ServiceClient) URL() string {
 	return s.generated().Endpoint()
 }
 
-// TODO: CreateQueue
-// TODO: DeleteQueue
-
 // GetProperties - gets the properties of a storage account's Queue service, including properties for Storage Analytics
 // and CORS (Cross-Origin Resource Sharing) rules.
-func (s *Client) GetProperties(ctx context.Context, o *GetPropertiesOptions) (GetPropertiesResponse, error) {
+func (s *ServiceClient) GetProperties(ctx context.Context, o *GetPropertiesOptions) (GetPropertiesResponse, error) {
```

Comment:
Can we rename the return type to `GetServicePropertiesResponse` so the pattern is consistent with the queues client.